internal/transport/handler_server.go

```diff
@@ -141,8 +141,13 @@ type serverHandlerTransport struct {
 	stats []stats.Handler
 }
 
-func (ht *serverHandlerTransport) Close() {
-	ht.closeOnce.Do(ht.closeCloseChanOnce)
+func (ht *serverHandlerTransport) Close(err error) {
+	ht.closeOnce.Do(func() {
+		if logger.V(logLevel) {
+			logger.Infof("Closing serverHandlerTransport: %v", err)
+		}
+		ht.closeCloseChanOnce()
+	})
```

Comment:
Let's do the actual `close` inline here and delete the other function.

Reply:
Yeah, good point. Deleted other function, moved inline.